### ISSUE TITLE
How to create your own software community

### ISSUE THREAD

**Issue by Bisaloo** [ropensci-org/community-calls#24]
# Topic

I thought about this while talking with @thomased about standards we would like to see spreading to the colour science community, and how colour science package developers could help one another in making sure the various packages stay available on CRAN.

Additionally, I recently read [RECON's guidelines to package development and onboarding process](https://www.repidemicsconsortium.org/resources/guidelines/) (the onboarding process part is not explicitly mentioned but @thibautjombart said in a private email packages can be transferred to @reconhub).

In both cases, it sounds like the goal is to build a rOpenSci-like community. I think rOpenSci has gathered enough maturity and experience to be able to give advice on how to set up "a software community" (for a lack of better word) and I'd love to see a call on this topic.

### Who is the audience?

I think the audience can be quite broad. I would say "package developers interesting in a specific area of science" / "researchers that develop packages" and would be interested in setting up their own community.

In terms of existing organisations, I think this could be particularly relevant for @reconhub.

### Why is this important?

rOpenSci has been doing an incredible job to improve the quality and sustainability of scientific software but the scope for submissions remains quite narrow and good-quality & useful packages might be rejected just because they don't fit rOpenSci's scope. To overcome this issue, I think it's time for [standards to go wild](https://ropensci.org/blog/2019/04/18/wild-standards/) and for rOpenSci to start clones in many areas of science.

### What should be covered?

- How to develop / choose good practice rules / standards?
- How to get people interested in the initiative?
- ???

In short, this could be seen as a "what have we learned until now?" for rOpenSci's staff.

### Suggested speakers or contributors

@mdpadge for the "how to develop a set of standards"?

Otherwise, the insights of most @ropensci's staff members would be relevant.

It might be nice to have some outside speakers as well(?) but I have no idea of who it could be at the moment.

**Comment by stefaniebutland** [comment]
I think this would be popular and beneficial to many. I have a couple of comm calls I *must* deliver in the first half of 2021 but will keep this in mind. Thanks for the details

**Comment by thibautjombart** [comment]
thanks @Bisaloo for the idea, I confirm that we at RECON would be very interested in this indeed :)

**Comment by stefaniebutland** [comment]
@lwasser sounds like this topic is up your alley for @pyOpenSci.

**Comment by lwasser** [comment]
it sure is thanks @stefaniebutland !! we are thinking a lot about how we can work with existing subdomains given so many exist in the python world and we can't do everything. I'd be interested in specifically what ROS doesn't cover that you need and/or what you could give to ROS and use from ROS as well! we will be developing a model with Pangeo for this type of thing IF we are funded :) 🤞  it seems like ROS and PyOS can offer a lot to domain specific communities that would avoid some of the pain points associated with starting from scratch.

**Comment by Rekyt** [comment]
One interesting point maybe to differentiate between two scenarios of package ecosystems:

1. When the packages are supposed to work **together** in succession in a workflow;
2. When the functionality of the different packages are nested within each other.

For example in the tidyverse `dplyr` and `tidyr` are following the first type of relationships, while `tibble` and `pillar` have "nested" functionalities and provide the base infrastructure to the other packages.

I am also thinking about the ecosystem built by the [folks at easystats](https://github.com/easystats) with many packages that work independently but in a coherent way (`parameters`, `performance`, `correlation`, and `see`) then they built a paramount package `report` that include functionalities from all the previous one.

I think this distinction may also help shape the discussion between having user-facing package ecosystems vs. package ecosystems that are meant for package developers only. We probably want to focus on the former.

**EDIT**: I realized that I am probably a little off-topic here, as I was more interested in talking about package ecosystems rather than package **communities**, but I think that thinking about how package are supposed to connect can still be relevant in this discussion

**Comment by stefaniebutland** [comment]
@asteiker I was just reminded of this thread and thought of you and the developing [earthaccess](https://earthaccess.readthedocs.io/en/latest/) community! Here to say the folks on this thread are awesome...like you. Great folks if you're ever looking to have a discussion. 
Sending ❤️ to everyone here.

**Comment by lwasser** [comment]
As always, you are so thoughtful @stefaniebutland  and such a graceful connector (which i always admire!) ❤️
